components/builder-api/habitat/config/config.toml

```diff
@@ -3,17 +3,20 @@ log_dir = "{{pkg.svc_var_path}}"
 [ui]
 root = "{{pkg.svc_static_path}}"
 
-{{toToml cfg}}
+[http]
+{{toToml cfg.http}}
+[web]
+{{toToml cfg.web}}
+[github]
+{{toToml cfg.github}}
 
-{{~#eachAlive bind.router.members}}
+{{~#eachAlive bind.router.members as |member|}}
 [[router]]
-host = "{{sys.ip}}"
-port = {{cfg.port}}
+host = "{{member.sys.ip}}"
```

Comment:
I much prefer using that yielded variable--makes things a lot clearer which `sys.ip` we're talking about. I learned something new today…